CHANGELOG.md

```diff
@@ -1,9 +1,26 @@
 
 
-## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1+1...master)
+## [Unreleased](https://git.openjdk.org/jtreg/compare/jtreg-7.1.1+1...master)
 
 * Improved support for JUnit Jupiter.
   * Update jtreg to bundle JUnit 5.9.2 [CODETOOLS-7903406](https://bugs.openjdk.org/browse/CODETOOLS-7903406)
+  * Improve reporting for errors in JUnit's lifecycle methods [CODETOOLS-7903430](https://bugs.openjdk.org/browse/CODETOOLS-7903430)
+
+* Improved support TestNG.
```

Comment:
insert _for_ : Improved support for TestNG

Reply:
```suggestion
* Improved support for TestNG.
```